PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCacheRenderers/MediaView.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  PBMMediaViewT.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import UIKit
+
+ let DEFAULT_VIEWABILITY_POLLING_INTERVAL: TimeInterval = 0.2
```

Comment:
Let's move out from naming in macro-style. Just use camelCase notation. Also it makes sense to mark such constants with `fileprivate`.